### PR TITLE
Return cipher as array

### LINKED ISSUE #29: Create a method for retrieving the encrypted data from the note
Currently only within the phoenix core library can the ciphers be called for the hashing method. 

For the method to be used elsewhere, we need to be able to call these fields when the library is exported.

### PR DESCRIPTION
**Pull request by LukePearson1**
Here we implement a function for returning the encrypted data as an array of BlsScalars

This closes #29

### CODE REVIEW

**Review by CPerezz — CHANGES_REQUESTED** [review]
The code addition is fine, but you'll need to bump the patch version of the crate otherways you'll not be able to use it.

File: src/note.rs [hunk 1]
```diff
@@ -261,6 +261,12 @@ impl Note {
         &self.value_commitment
     }
 
+    /// Returns the encryped data from the note as an array of
+    /// BlsScalar's
```
Comment (ZER0) [inline src/note.rs:265]: This should be more like `Returns the encrypted data's cipher` or something like that.

File: src/note.rs [hunk 2]
```diff
@@ -261,6 +261,12 @@ impl Note {
...
+    pub fn cipher(&self) -> [BlsScalar; CIPHER_SIZE] {
+        *self.encrypted_data.cipher()
```
Comment (ZER0) [inline src/note.rs:267]: This should match the signature of the actual `cipher` method that we're proxying, so it should be: 
```rust
pub fn cipher(&self) -> &[BlsScalar; CIPHER_SIZE] {
    self.encrypted_data.cipher()
}
```

**Review by ZER0 — CHANGES_REQUESTED** [review]
Small nits, mostly about the function's signature

**Review by CPerezz — APPROVED** [review]
LGTM. Remember to release a new version in GH.

**Pull request merged**